util/symbols.py

```diff
@@ -1,12 +1,14 @@
 from dataclasses import dataclass
 from typing import Dict, List, Optional
+import spimdisasm
 
-from capstone import CsInsn
 from util import options, log
 
 all_symbols: "List[Symbol]" = []
 symbol_ranges: "List[Symbol]" = []
 sym_isolated_map: "Dict[Symbol, bool]" = {}
+# Initialize a spimdisasm context, used to store symbols and functions
+spim_context: spimdisasm.common.Context = spimdisasm.common.Context()
```

Comment:
```suggestion
spim_context = spimdisasm.common.Context()
```
unless it does serve a purpose?